client/src/app/site/pages/meetings/pages/meeting-settings/pages/meeting-settings-group-detail/components/meeting-settings-group-detail-field/meeting-settings-group-detail-field.component.ts

```diff
@@ -226,7 +226,7 @@ export class MeetingSettingsGroupDetailFieldComponent extends BaseComponent impl
                 }
             ]
         });
-        if (this.disabled) {
+        if (this.disabled || this.getWarning()) {
```

Comment:
please change the method title to sth more appropriate for a boolean return value (e.g. hasWarning)